HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -4688,6 +4688,26 @@ Patch "Infinite Quick Cyloop Gauge" in "Cheats" by "NM"
     );
 }
 
+Patch "Play Monologues Unconditionally" in "Audio" by "WasifBoomz" does
+/*
+The random voice clips around the island are played no matter what.
+
+Previously, they would not play if Sonic was not on regular terrain.
+*/
+{
+    WriteProtected<byte>
+    (
+        /* 0x1401CA014 */
+        ScanSignature
+        (
+            "\x84\xC0\x74\x0E\x8B\x47\x48",
+            "xxxxxxx"
+        ),
+
+        Assemble("test al, 2")
```

Comment:
For future reference (you don't have to change this); when changing a condition, you should modify the jump instruction instead of the comparison unless you really have to.

I'm not seeing a result from the register you're testing where it'd ever be 2, so you could simply change the jump instruction at `0x1401CA016` to `jmp` instead of `jz` - the opcode for which is just `0xEB`.

For cases where you'd be changing jumps, write the bytecode directly, but for all other instructions, stick to `Assemble()`.